### DISCUSSION TITLE
VSCode extension for CodeIgniter 4

### DISCUSSION THREAD

**Discussion by Sunchock** [codeigniter4/devkit#133] [General]
Hi,

I would like to know if there is a plan to create a VSCode extension to support CI4, like https://github.com/smallp/vscode-ci does for CI3 please?

**Comment by datamweb** [comment]
I haven’t read the documentation for the extension in detail, but it seems that there might not be a need for a specific CI4 extension. CodeIgniter 4 heavily utilizes phpdocs, so by installing the [PHP Intelephense extension](https://marketplace.visualstudio.com/items?itemName=bmewburn.vscode-intelephense-client) and opening your project in VSCode, you should get similar functionality to what you’re looking for.
